apiml/src/test/java/org/zowe/apiml/acceptance/AcceptanceTest.java

```diff
@@ -27,15 +28,23 @@
 
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
-@ComponentScan(basePackages = "org.zowe.apiml", excludeFilters = {
-    @ComponentScan.Filter(
-        type = FilterType.REGEX,
-        pattern = ".*Application"
-    ),
-    @ComponentScan.Filter(
-        type = FilterType.ASSIGNABLE_TYPE,
-        classes = EurekaController.class
-    )})
+@ComponentScan(
+    basePackages = "org.zowe.apiml",
+    excludeFilters = {
+        @ComponentScan.Filter(
+            type = FilterType.REGEX,
+            pattern = ".*Application"
```

Comment:
Can we be more specific and provide `ASSIGNABLE_TYPE` with a list of main classes?

Reply:
We'll need to remember to update the list in each subsequent PR that includes a new module